src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -20,7 +20,7 @@
   /** Creates a new Climber. */
   public Climber() {
     m_extensionController = new TalonFX(Constants.ClimberExtension);
-    m_solenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.SolenoidForward, Constants.SolenoidReverse);
+    m_solenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.ClimberSolenoidForward, Constants.ClimberSolenoidReverse);
```

Comment:
We found out tonight that we're using the REV pnumiatics hub, not CTRE as we originally thought.

Reply:
We did this